fuelup-init.sh

```diff
@@ -25,8 +24,7 @@ main() {
     mkdir -p "$FUELUP_DIR/bin"
 
     local _fuelup_version
-    _fuelup_version="$(curl -s https://api.github.com/repos/FuelLabs/fuelup/releases/latest | jq -r ".tag_name")"
-    _fuelup_version="$(echo "${_fuelup_version}" | cut -c 2-)"
+    _fuelup_version="$(curl -s https://api.github.com/repos/FuelLabs/fuelup/releases/latest | grep -r "tag_name" | cut -d "\"" -f4 | cut -c 2-)"
```

Comment:
```suggestion
    _fuelup_version="$(curl -s https://api.github.com/repos/FuelLabs/fuelup/releases/latest | grep "tag_name" | cut -d "\"" -f4 | cut -c 2-)"
```
Just tested this locally, but had to remove the recursive flag to get this to work. Otherwise looks good!